examples/tools/scene_viewer.rs

```diff
@@ -260,58 +217,26 @@ fn camera_spawn_check(
         let size = (max - min).length();
         let aabb = Aabb::from_min_max(Vec3::from(min), Vec3::from(max));
 
-        if !scene_handle.has_camera {
-            let bundle = if let Some((transform, camera)) = cameras_3d.iter().next() {
-                let mut transform: Transform = (*transform).into();
-                let (yaw, pitch, _) = transform.rotation.to_euler(EulerRot::YXZ);
-                transform.rotation = Quat::from_euler(EulerRot::YXZ, yaw, pitch, 0.0);
-                PerspectiveCameraBundle {
-                    camera: camera.clone(),
-                    transform,
-                    ..PerspectiveCameraBundle::new_3d()
-                }
-            } else {
-                let transform = Transform::from_translation(
+        info!("Spawning a controllable 3D perspective camera");
```

Comment:
Hmm, I'm not so keen on not using one of the cameras from the scene by default...

Reply:
It does actually use cameras from the scene by default (and that was the intent). Just tested with a simple scene exported from blender. However I think this relies on "undefined behavior", specifically that the camera_tracker Query iterates entities in "spawn order" and sets the first matching camera to active. The scene is always spawned before the "controlled camera", so this works.  I doubt this will change any time soon, but it would be good to not rely on that behavior, as we explicitly tell users not to rely on it.

Reply:
Just changed this to explicitly prefer scene cameras.